Source/WebCore/loader/cache/CachedFont.cpp

```diff
@@ -64,10 +68,47 @@ void CachedFont::didAddClient(CachedResourceClient& client)
         downcast<CachedFontClient>(client).fontLoaded(*this);
 }
 
+static AtomString hashForFontData(const uint8_t* data, size_t size)
```

Comment:
We should take a `Span<const uint8_t>` in modern code instead of raw pointer + size.